content/200-concepts/100-components/02-prisma-client/240-metrics.mdx

```diff
@@ -20,23 +20,25 @@ Prisma Client provides the following metrics:
 
 - Counters (always increase):
 
-  - `query_total_queries`: The total number of database queries executed.
-  - `query_total_operations`: The total number of Prisma operations executed.
-    - A Prisma operation is a statement like `await prisma.user({ include: { posts: true }})`, which Prisma Client translates into one or more SQL queries.
-    - The value returned by `query_total_operations` can be less than `query_total_queries`, because some operations create multiple queries.
+  - `prisma_datasource_queries_total`: The total number of datasource queries executed.
+  - `prisma_client_queries_total`: The total number of Prisma Client queries executed.
+    - The value returned by `prisma_client_queries_total` can be less than `prisma_datasource_queries_total`, because some Prisma Client operations create multiple queries.
+  - `prisma_pool_connections_opened_total`: The total number of [pool connections](/concepts/components/prisma-client/working-with-prismaclient/connection-pool#default-connection-pool-size) opened.
+  - `prisma_pool_connections_closed_total`: The total number of pool connections closed.
 
 - Gauges (can increase or decrease):
 
-  - `pool_active_connections`: The number of active connections in use.
-  - `pool_idle_connections`: The number of connections that are not being used.
-  - `pool_wait_count`: The number of workers waiting for a connection because all connections are in use.
-  - `query_active_transactions`: The number of active transactions.
+  - `prisma_pool_connections_busy`: The number of currently busy pool connections. These pool connections are currently executing a datasource query.
+  - `prisma_pool_connections_idle`: The number of pool connections that are not currently being used. These pool connections are waiting for the next datasource query to run.
+  - `prisma_pool_connections_open`: The number of currently open pool connections.
+  - `prisma_client_queries_wait`: The number of Prisma Client queries currently waiting for a connection because all connections are in use.
+  - `prisma_client_queries_active`: The number of currently active transactions.
```

Comment:
Changed order:
- Open is superset/sum of busy and idle
- Active is the more normal than wait for queries

```suggestion
  - `prisma_pool_connections_open`: The number of currently open pool connections.
  - `prisma_pool_connections_busy`: The number of currently busy pool connections. These pool connections are currently executing a datasource query.
  - `prisma_pool_connections_idle`: The number of pool connections that are not currently being used. These pool connections are waiting for the next datasource query to run.
  - `prisma_client_queries_active`: The number of currently active transactions.
  - `prisma_client_queries_wait`: The number of Prisma Client queries currently waiting for a connection because all connections are in use.
```